website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,682 @@
+
+  if (!extension || !isOpen) return <></>;
+
+  const getInstallUrl = (extension: ProcessedExtension): string => {
+    return `podman-desktop:extension/${extension.id}`;
+  };
+
+  return (
+    <div
+      className="fixed inset-0 flex items-center justify-center z-250 p-4"
+      style={{ backgroundColor: 'rgba(0, 0, 0, 0.85)' }}
+      onClick={onClose}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)